app/views/shared/transactions/_search2.html.haml

```diff
@@ -1,6 +1,6 @@
 -# This view, replaces _search, which relies on the TransactionSearch controller/concern.
 -# This view, along with TransactionSearch::Searcher is the more modern method. See FacilityAccountsReconciliationController
-= simple_form_for @search_form, url: url_for(params), method: :get, html: { class: "search_form" }, defaults: { required: false } do |f|
+= simple_form_for @search_form, url: url_for(params.permit!), method: :get, html: { class: "search_form" }, defaults: { required: false } do |f|
```

Comment:
So, turns out `permit(:search)` did not in fact work. Are you opposed to `permit!` here? It seems reasonably safe, since we're effectively filtering out only the search params for which we already have a searcher defined on whichever particular controller action.

Reply:
I was just trying it out. It looks like we might be able to get away with just `url: url_for,`. The empty `url_for` will link to the current path with controller/action. I'm pretty sure whatever query params are included in this URL are lost anyways.

I know we now maintain the filter params in the sort headers (#1466), but if we want to maintain the sort after filtering, then I think some hidden fields are probably the way to go.

Speaking of, how does the change in #1466 not get flagged by Rails 5?

Reply:
I think #1466 might actually get flagged by Rails 5, but I figure we'll cross that bridge when we get there.